sdc/tests/test_series.py

```diff
@@ -2977,6 +3011,51 @@ def test_impl(S):
         S = pd.Series([np.nan, 2., 3., np.inf])
         pd.testing.assert_series_equal(hpat_func(S), test_impl(S))
 
+    def test_series_isna_index(self):
```

Comment:
You could combine the test with `test_series_isna_str_index` through the loop.

Reply:
Done